### DISCUSSION TITLE
大家去過夜店嗎

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2271] [General]
我身為一個大三生，到現在都還沒去過夜店，不過主要是身邊的同學不是有女朋友就是不敢社交，所以沒人要去，我其實也不懂我這種大方的人為啥都沒啥社交的感覺，是不是不應該把人生都花在研究學習創業啊，好像我人生只有這三件事，超無聊哈哈哈，雖然去夜店也不一定有社交，並經我不到170，根本沒人要吧哈

**Comment by Teddy111802518** [comment]
夜店確實是大型社交場所，超好笑我還天真地以為大家是去交朋友的，結果我同學直接戳破，說夜店交朋友的少之又少，大家都是找人約跑步的，害我覺得我其實也不適合去，頂多去看有沒有kpop之夜去給他跳個舞囉

**Reply by kzx-creat** [reply]
我覺得約跑步應該是極少數，感覺多數都是去喇跟認識別人，搜尋約跑步潛力對象?

**Comment by zoo100130** [comment]
我也老人了到現在還沒去過夜店，身邊朋友也都那種乖乖讀書的，但有時就希望能有一些有活力的社交，看IG推日落俱樂部、夜店推薦廣告也沒人去(但我酒量hen差94)
![image](https://github.com/user-attachments/assets/516333b3-9ea0-4c96-a582-493abc43c7db)

**Comment by kzx-creat** [comment]
Q我也沒去過夜店，大家要在交男女朋友之前去個幾次才對(沒亂講的。話說，沒170不會沒人要啦，愛你的人不會只看到你的身高。

**Reply by zoo100130** [reply]
好奇問個，那如果錢包裡面沒1700呢

**Reply by kzx-creat** [reply]
沒事，我也沒1700

**Reply by zoo100130** [reply]
你要去的話我可以跟你一起去刷成就🙂

**Comment by christaaah** [comment]
完全沒去過，也不想去，因為我超級社恐，也不太擅長應付社交的場合，最重要的是我不喝酒，所以夜店完全沒有吸引我的地方，比起去夜店我比較喜歡待在宿舍玩遊戲、上網或睡覺，對我來說是最舒適的，但是如果是喜歡社交的人去應該會很享受吧。

**Comment by 113607514** [comment]
其實我也常覺得，自己生活好像就只剩下學習、研究和創業這三件事，感覺有點單調。雖然不去夜店也不代表沒社交，但有時候會覺得人生好像缺少了一點不同的色彩，雖然說到夜店，自己不到170cm，感覺也不太可能成為焦點，哈哈。

**Reply by Teddy111802518** [reply]
你的創業內容是什麼，雖然我覺得妳只是改寫我的文案哈哈

**Comment by Taoyi-Chiang** [comment]
夜店滿好玩的啊，就是大家喝酒跳舞這樣，至於要不要約跑就看個人吧。
我自己是有認識滿多有趣的人，滿突破同溫層交友圈的。

**Comment by Elaine4488** [comment]
沒去過，好像也不會特別想去欸。只有看過朋友限動，聽說是有些玩得很嗨尺度很大，但也有相較於單純的環境 (?
